src/utils/Logger.js

```diff
@@ -1,19 +1,44 @@
 import colors from 'colors'
 
+const defaults = {
+  verbose: false,
+  silent: true
+};
+
 export default class Logger {
-  constructor(prefix) {
+  constructor(prefix, opts) {
     this.prefix = prefix
+    this.opts = Object.assign({}, defaults, opts);
   }
 
   out(msg) {
     console.log(msg)
   }
 
   info(msg) {
-    console.error(`[${this.prefix}] ${msg}`.green)
+    this.log(msg, 'green')
   }
 
   error(msg) {
-    console.error(`[${this.prefix}] ${msg}`.red)
+    this.log(msg, 'red')
+  }
+
+  log(msg, color) {
+    if (this.opts.silent) {
+      return;
+    }
+    if (this.opts.verbose) {
+      console.error(`[${this.prefix}] ${msg}`[color])
+    } else {
+      console.error(msg[color])
+    }
+  }
+
+  static silent(value) {
+    defaults.silent = value;
+  }
+
+  static verbose(value) {
+    defaults.verbose = value;
   }
-}
+}
```

Comment:
I'd simply move the static methods above the constructor since these are the entry-points of this object